paper.tex

```diff
@@ -600,13 +600,13 @@ \subsubsection{Growth of the Unit}
 \subsection{Outsourcing}
 
 Another possibility for the realisation of local RSE Service providers is by forming a spin-off and pooling the RSE Skills into an external company, which has benefits but also drawbacks.
-This is an idea, that so far lacks examples but for completeness we list some of the advantages and disadvantages.
+This is an idea that so far lacks examples but for completeness we list some of the advantages and disadvantages.
 Among the most obvious benefits is that this enables the creation of contracts outside of the WissZeitVG.\@
 This also widens the customer base of the RSE unit since the newly founded company may obtain contracts from industry.
 If this company is backed/branded by the institution, this enables another possibility for an institution to interact with local companies.
 This might open opportunities for employees to more freely move between industries and academia.
 But there are drawbacks.
-Since the company is now an entity external to the institution, the Vergabe-Richtlinien have to be fulfilled, which could \eg{} mean to publicly invite tenders in order to have a competitive procedure.
+Since the company is now an entity external to the institution, the Vergaberichtlinien have to be fulfilled, which could, \eg{} mean to publicly invite tenders in order to have a competitive procedure.
```

Comment:
```suggestion
Since the company is now an entity external to the institution, the “Vergaberichtlinien” (procurement guidelines) have to be fulfilled, which could, \eg{} mean to publicly invite tenders in order to have a competitive procedure.
```